tests/phpunit/integration/Modules/AdSenseTest.php

```diff
@@ -204,6 +205,10 @@ public function test_block_on_consent_amp_content( $enabled ) {
 			add_filter( 'googlesitekit_adsense_tag_amp_block_on_consent', '__return_true' );
 		}
 
+		// We need to fake the global to allow the hook to add the tag.
+		global $wp_query;
+		$wp_query->in_the_loop = true;
```

Comment:
While this is necessary to make this test pass, let's add another test for this specifically to test that the tag is also not added when not in the loop.

Reply:
This test is yet to be added it seems.